### DISCUSSION TITLE
For POJOs with Boolean type of variables, Can we have the getter as "is" instead of "get"

### DISCUSSION THREAD

**Discussion by siva54** [kobylynskyi/graphql-java-codegen#932] [Q&A] [answered]
For example,  We have something like this in the schema,

`updated: Boolean`

in the schema, then the POJO with the getter is available as,

`public Boolean getUpdated()`

Is it possible to have it this way?

`public Boolean isUpdated()`

Since this is nicer to read for a Boolean/boolean type.

**Comment by kobylynskyi** [comment, marked as answer]
hey @siva54 good point,
this can be done, but in the future major release because it is a breaking change

**Reply by siva54** [reply]
Thank you @kobylynskyi
